management/version.go

```diff
@@ -0,0 +1,5 @@
+package management
+
+var (
+	sdkVersion = "8.0.0-beta"
```

Comment:
Any changes in the update script we need to make so this is updated automatically?  I know that ASM doesn't change much so it's not really high on the priority list.